fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/NavigationBar.kt

```diff
@@ -180,14 +185,26 @@ object NavigationItems {
 @Composable
 private fun NavigationBarPreview() {
     FirefoxTheme {
-        NavigationBar(NavigationItems.defaultItems)
+        NavigationBar(
+            NavigationItems.defaultItems,
+            browserStore = BrowserStore(),
+            browsingModeManager = object : BrowsingModeManager {
+                override var mode: BrowsingMode = BrowsingMode.Normal
+            },
+        )
     }
 }
 
 @Preview
 @Composable
 private fun NavigationBarPrivatePreview() {
     FirefoxTheme(theme = Theme.Private) {
-        NavigationBar(NavigationItems.defaultItems)
+        NavigationBar(
+            NavigationItems.defaultItems,
+            browserStore = BrowserStore(),
+            browsingModeManager = object : BrowsingModeManager {
+                override var mode: BrowsingMode = BrowsingMode.Private
```

Comment:
why default to private?

Reply:
Because it's a preview for the private theme, isn't it :)

Reply:
d'oh! didn't see that